redi-connect-infrastructure/README.md

```diff
@@ -2,91 +2,123 @@
 
 In this project we keep all the infrastructure redi-connect needs to run. We do that writing infrastructure as code using [terraform](https://www.terraform.io/).
 
-## Prerequisites
-
-The first time one executes terraform on an azure subscription they would have to create an azure container where they can store terraform state.
+## Prerequisites & Getting started
 
+Before using using Terraform on an Azure subscription, an Azure container is required to store Terraform state.
 
 This is the list of prerequisites required for the `connect` ci/cd pipeline:
 
-1. Before you run `terraform init` - Create the backend for azure.
-
-    resource:  https://docs.microsoft.com/en-us/azure/developer/terraform/store-state-in-azure-storage
-    
-    We do this using terraform. The tf code to provision the backend is in the `az-remote-backend-*` files.
-    When you `terraform aply` for the first time the backend will be provisioned for you in azure.
-    Also you will see the outputs, something like:
-    
-    ```
-    storage_account_name: tstate8694
-    container_name: tstate
-    access_key: ****
-    
-    ``` 
-    You will have to copy the `storage_account_name: terraformstateXXXX` and paste them in the `_versions.tf` file like so
-    ```
-     backend "azurerm" {
-       storage_account_name  = "terraformstateXXXX"
-        .
-        .
-        .
-     }
-    ```
-    
-    Then you can run `terraform init`
-    
+1. Create the backend for Azure.
+
+   This is step is needed only the very first time _before_ you run `terraform init`. This is because when you are under a completely new Azure subscription you dont have a storagr container to store the Terraform state yet.
+   
+   Resource: https://docs.microsoft.com/en-us/azure/developer/terraform/store-state-in-azure-storage
+
+   The very first time in order to run `terraform init` we need storage where we can store the state of Terraform. To do so you will simply need to execute the `generate-tf-storage.sh` script.
+
+   From the directory `redi-connect-infrastructure` run
+   ```
+   ./scripts/generate-tf-storage.sh
+   ```
+
+   This will provision an Azure container for you in Azure.
+
+   In your terminal you will also see some outputs generated. Something like:
+
+   ```
+   storage_account_name: tstate8694
+   container_name: tstate
+   access_key: ****
+
+   ```
+
+   Copy the `storage_account_name: terraformstateXXXX` and paste them in the `_versions.tf` file like so
+
+   ```
+    backend "azurerm" {
+      storage_account_name  = "terraformstateXXXX"
+       .
+       .
+       .
+    }
+   ```
+   You can now run `terraform init`
```

Comment:
```suggestion
   Now, run `terraform init`
```

Reply:
I am in favour for politeness but I will push this one ;)